tron/core/job_scheduler.py

```diff
@@ -195,6 +195,11 @@ def get_runs_to_schedule(self, ignore_last_run_time):
         next_run_time = self.job.scheduler.next_run_time(last_run_time)
         return self.job.build_new_runs(next_run_time)
 
+    def update_name(self, name):
```

Comment:
I wonder if you need this. You could just set the name in JobCollection.move and call schedule_reconfigured afterwards.

Reply:
The reason I didn't call schedule_reconfigured is the variable, "ignore_last_run_time", should be False when we move the job. I can either keep update_name function here or refactor schedule_reconfigured function. No preference.